.docker-mongo/Dockerfile

```diff
@@ -36,12 +36,12 @@ RUN aptMark="$(apt-mark showmanual)" \
     && apt-mark auto '.*' > /dev/null \
     && apt-mark manual $aptMark > /dev/null \
     && find /usr/local -type f -executable -exec ldd '{}' ';' \
```

Comment:
"on behalf of" definitely not right 😬

Reply:
I have to say I don't know what that does.. it was copied from node's Dockerfile https://github.com/nodejs/docker-node/blob/66b46292a6e5dd5856b1d5204dc51547c80eb17a/14/buster-slim/Dockerfile#L45 🤷

Reply:
Looks to me like it's making sure all npm/node dependencies are not accidentally removed when removing some other package with a similar dependency, thus not reducing/eliminating child image issues.

Also, looking at the Dockerfile, I'm not sure we even need those lines to be part of our image, do you remember why those were added initially? 🤔

Reply:
this image specifically is not intended for production, we call is as "preview" since it has mongo installed and can be used to have a "preview" of a rocket.chat version.

we use the same technic on the production one though.. the ideia is to install some packages just to run the `npm install` and then clean it up..

Reply:
Hmm, looks like I commented on the wrong file, but these are part of our production one as well `.docker/Dockerfile`.

this isn't a big deal 🙈 maybe slightly more time spent on 'building', nothing else.